README.md

```diff
@@ -36,3 +36,10 @@ jobs:
     # Required to pass along secrets for `CLA_BOTIFY_TOKEN`
     secrets: inherit
 ```
+
+## Rulesets
+GitHub [org-level Rulesets](https://docs.github.com/en/enterprise-cloud@latest/repositories/configuring-branches-and-merges-in-your-repository/managing-rulesets/available-rules-for-rulesets#require-workflows-to-pass-before-merging) can be configured to run a workflow check against pull requests in all repos in the org. This is a very powerful feature, but there are some caveats and best practices to be aware of when enabling a Ruleset.
+
+- Supported Event Triggers are documented [here](https://docs.github.com/en/enterprise-cloud@latest/repositories/configuring-branches-and-merges-in-your-repository/managing-rulesets/available-rules-for-rulesets#supported-event-triggers). However, when a workflow runs in response to a Ruleset, some configs such as `branches`, `paths`, `paths-ignore`, that would normally be valid in a workflow are ignored. If you need to target or exclude specific branches, that can be configured in the Ruleset settings. If you need to target or exclude specific paths, that must be implemented manually in the workflow itself.
```

Comment:
Probably worth mentioning that selected actions are ignored and only the default actions for the given type are used. NAB consider breaking into sub-bullets for readability